mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOpInterfaces.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) Intel
+
+//===--- SYCLOpInterfaces.h -----------------------------------------------===//
+//
+// MLIR-SYCL is under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MLIR_SYCL_OPS_INTERFACES_H_
+#define MLIR_SYCL_OPS_INTERFACES_H_
+
+#include <type_traits>
+
+#include "mlir/Dialect/SYCL/IR/SYCLOpsTypes.h"
+
+#include "mlir/Dialect/SYCL/IR/SYCLOpInterfaces.h.inc"
```

Comment:
[minor]: remove empty lines between headers.

Reply:
https://github.com/intel/llvm/pull/6934#discussion_r985958508

Cannot remove due to order requirement and clang-format.